en/ch09.md

```diff
@@ -35,16 +35,25 @@ The `Interface` tab is where you can adjust appearance and security settings of
 (4) **Bootstrap Address:** Enter the hostname or IP address of the bootstrap remote node. See section [Bootstrap nodes](#bootstrap-nodes) for a brief explanation of what a bootstrap node is.  
 (5) **Bootstrap Port:** Enter the port of the bootstrap remote node.  
 
-### Remote Node
-Use a remote node, do not download the blockchain. Check the ['About remote nodes' section](#about-remote-nodes) of this guide.
+### Node: Remote node
+Select this option if you want your Monero GUI to connect to the Monero network using a remote node, and if you don't want to download the whole blockchain on the current computer. Remote nodes are servers running local nodes that have a full blockchain. These servers are usually run by third parties (which can be dangerous to your privacy), but you can also run your own remote node if you have technical knowledge. Check the ['About remote nodes' section](#about-remote-nodes) of this guide.
```

Comment:
` Remote nodes are servers running local nodes that have a full blockchain.` ->Remote nodes could be running pruned blockchains as well.